.gitignore

```diff
@@ -192,3 +192,4 @@ Temporary Items
 
 #Visual studio stuff
 *.vscode/*
+/config/admins.txt
```

Comment:
this shouldn't be in gitignore

Reply:
Can you advise me,how to deal with admins.txt ?

Reply:
revert all changes you've made to it and just add yourself to the list normaly